Slim/Player/HTTP.pm

```diff
@@ -64,7 +68,6 @@ sub resume      { 1 }
 sub volume      { 1 }
 sub fade_volume { 1 }
 sub bufferFullness { 0 }
-sub formats     { 'mp3' }
```

Comment:
Should we define a default value here?

Reply:
It was in Web::HTTP but yes, it's a better practice to set it inside the player